### ISSUE TITLE
Bewertungsansicht für Teilnehmer sehr zerfleddert

### ISSUE THREAD

**Issue by hermannkm** [dasistwas/moodle-theme_campus#1]
![bildschirmfoto 2015-06-13 um 07 05 30](https://cloud.githubusercontent.com/assets/12869273/8143109/5d34922a-119b-11e5-8f3b-6d8d2a1c42aa.png)
Wie man sieht zerstört die schmale Spaltenbreite links die Seitenansicht.

**Comment by hermannkm** [comment]
Just another example and the problem description in English (don't know who is reading this): the user report for my grading (which is what my pupils will see) is almost unreadable because of very narrow columns. Don't know how to adjust them (or get rid of the explaining text). As no docking is possible for the menus I don't see how to solve this problem.
![bildschirmfoto 2015-06-13 um 16 49 29](https://cloud.githubusercontent.com/assets/12869273/8144740/4a27f3ec-11ec-11e5-8c23-85ea91513588.png)

**Comment by gjb2048** [comment]
Hi,

I'm the contracted developer who wrote the theme.  I've contacted my client to ask about how I proceed with this.  It's related to this code: https://github.com/dasistwas/moodle-theme_campus/blob/master/less/campuscustom.less#L1436-L1437 and as Campus was created from Shoelace: https://github.com/gjb2048/moodle-theme_shoelace/blob/master/less/shoelacecustom.less#L36-L37 which in turn is related to this: https://moodle.org/mod/forum/discuss.php?d=279883 and https://moodle.org/mod/forum/discuss.php?d=274915 and https://github.com/gjb2048/moodle-theme_shoehorn/issues/5.  Therefore a specific 'SCORM' targeting selector might be appropriate.

But I am unable to proceed without my client letting me know how they wish to proceed.

There is no 'docking' as that was a requirement of the theme.

Kind regards,

Gareth

**Comment by gjb2048** [comment]
P.S.

The problem is really down to the use of tables rather than 'div's in core Moodle which are harder to manipulate.  The snag is with the Clean theme this is not seen because there is no border: https://moodle.org/mod/forum/discuss.php?d=274915#p1184952

**Comment by hermannkm** [comment]
Hi Gareth,
Thank you for your reply. Please let me know if I can assist in any way - I like this theme very much. Btw I don't like the docks in moodle -  I just thought of how to get more space. There is no full screen mode for this view?

**Comment by gjb2048** [comment]
Thanks Hermann.  Looking at the navigation bar there is a full screen option.

**Comment by hermannkm** [comment]
Hi, I can see the icon for full screen but many times (including this time) it does not work - just no effect. Tried in Safari and Firefox on OSX 10.10. It does work sometimes.

**Comment by gjb2048** [comment]
Works for me every time on Chrome, Firefox and even Internet Explorer on Windows.  I don't have OSX upon which to test, only iOS for which would take me time (not now) to setup and test.

**Comment by hermannkm** [comment]
Just a short notice: tested on win8.1 (chrome and ie), on iOS8.3 (safari) and on my OS X 10.10 safari, Firefox and chrome: on ALL systems/browsers toggle full screen IS working flawlessly with a fresh install of moodle 2.9/campus, while my customized version does not. So I have to find out which settings I changed to break this function :-(. Have a nice weekend, more next week

**Comment by gjb2048** [comment]
Thanks.

Umm, I've not finished the M2.9 version yet.

**Comment by gjb2048** [comment]
Also...
1. I need to consult with my client about supporting the theme here.
2. Supporting a 'customised' version of a user of the theme is also another aspect that I need to discuss with my client.

**Comment by hermannkm** [comment]
Sorry for bothering you with 2.9 issues - realized only after doing these tests that campus is not released for 2.9. I will wait with further posts.

Hermann

> Am 14.06.2015 um 15:18 schrieb Gareth J Barnard notifications@github.com:
> 
> Thanks.
> 
> Umm, I've not finished the M2.9 version yet.
> 
> Sent from my iPod in deep space...
>

**Comment by dasistwas** [comment]
Hi,

thank you for discussing this issue. We will fix that for 2.9 release. 

Best,
David

**Comment by gjb2048** [comment]
Fixed in https://github.com/dasistwas/moodle-theme_campus/commit/8c68029db57e6014336d62904bc1e03bf89e252e.
